dialogue-core/src/test/java/com/palantir/dialogue/core/CoarseExponentialDecayReservoirTest.java

```diff
@@ -42,9 +42,9 @@ void testDecay_byHalf() {
         decay.update(2);
         assertThat(decay.get()).isEqualTo(2);
         clock.set(10);
-        assertThat(decay.get()).isOne();
+        assertThat(decay.get()).isBetween(0.9999, 1.0);
```

Comment:
Could use something like `assertThat(decay.get()).isEqualTo(1, Offset.offset(.0001));` I think `isEqualTo` with an offset is an alias to `isCloseTo`.